src/pudl/metadata/fields.py

```diff
@@ -535,6 +562,11 @@
         "type": "string",
         "description": "Account number, from FERC's Uniform System of Accounts for Electric Plant. Also includes higher level labeled categories.",
     },
+    "ferc_acct_name": {
+        "type": "string",
+        "description": "Name of FERC account, derived from technology description and prime mover code.",
+        "constraints": {"enum": ["Hydraulic", "Nuclear", "Steam", "Other"]},
```

Comment:
Is there a reason to go with the Title Case values here rather than standardizing to lower case like we do almost everywhere else? Are we not normalizing strings in this column? Where does it come from?

Reply:
These values are read in in `pudl.helpers.get_eia_ferc_acct_map()`, which reads in `pudl.package_data.glue.ferc_acct_to_pm_tech_map.csv`. In this CSV all fields (`technology_description`, `prime_mover_code`, `ferc_acct_name`) use Title Case. This table is then merged with the MCOE table to create the mega generators table. TitleCase is maintained in the merge keys and merged on fields in `pudl.analysis.plant_parts_eia.get_gens_mega_table()`. Since all of the fields use TitleCase (and a few others in the plant parts list/MCOE table, I'm leaving this as is for now.